CMakeLists.txt

```
@@ -43,10 +43,12 @@ include(GNUInstallDirs)
 include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/common.cmake)
 include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/aft.cmake)
 
+set(CMAKE_GENERATED_COMMENT "This file was auto-generated. DO NOT EDIT")
```

```suggestion
set(CMAKE_GENERATED_COMMENT "This file was auto-generated during CMake configuration, from a corresponding *.in file. DO NOT EDIT")
```

If anyone is reading this comment, and surprised by it, we probably want to send them towards the original source + generation process. I can't see a CMake variable for "source file currently being configured", so I think we put something generic like `*.in`.